interface/VideoEncoderDefs.h

```diff
@@ -164,6 +169,7 @@ typedef struct VideoRateControlParams {
     int8_t diffQPIP;// P frame qp minus initQP
     int8_t diffQPIB;// B frame qp minus initQP
     uint32_t layerBitRate[32]; // specify each scalable layer bitrate
+    EncMiscParameterHRD encMiscParameterHRD;
```

Comment:
I know this, ;) but I see duplicat code here.  whe have exactly same thing in VideoParamsHRD.
```
typedef struct VideoParamsHRD {
    uint32_t size;
    uint32_t bufferSize;
    uint32_t initBufferFullness;
}VideoParamsHRD;
```